awx/main/access.py

```diff
@@ -861,6 +862,34 @@ def can_unattach(self, obj, sub_obj, relationship, *args, **kwargs):
         return super(OrganizationAccess, self).can_attach(obj, sub_obj, relationship, *args, **kwargs)
 
 
+class HostMetricAccess(BaseAccess):
+    """
+    - I can see host metrics when I'm a super user or system auditor.
+    - I can delete host metrics when I'm a super user.
+    """
+
+    model = HostMetric
+
+    def get_queryset(self):
+        if self.user.is_superuser or self.user.is_system_auditor:
+            qs = self.model.objects.all()
+        else:
+            qs = self.filtered_queryset()
+        return qs
+
+    def can_read(self, obj):
+        return bool(self.user.is_superuser or self.user.is_system_auditor or (obj and obj.user == self.user))
```

Comment:
What do you think `obj.user` is supposed to be here? I don't see it as a field on the `HostMetric` model.

Reply:
Yeah, I was also wondering if it's required. I just followed the pattern below => I'll remove it